src/index.css

```diff
@@ -86,10 +87,52 @@ h6 {
   line-height: 1.2;
 }
 
+/*
+  Our implementation of DLs assumes you will wrap each DT/DD pair in a
+  wrapper DIV with the class name `.dli`, e.g.:
+  <dl>
+    <div class="dli">
```

Comment:
why doesn't the html spec just have li as a valid child of dl?

Please open a proposed change to the HTML spec thanks.

Reply:
Oh actually, and please boo me off the stage: would it be worth make a dli component so you can just do `<dl><dli>...` instead of the div class blah blah blah?

You could have that component expect / require content for the dd and the dt.  You'd be able to throw out this comment I think, and you'd also be able to avoid repetition of the class stuff in your dl implementations.

Reply:
Oh, I like that idea! I wonder if I can lowercase it though so it doesn't look weird between `<dl>` and `<dd>`…

> You could have that component expect / require content for the dd and the dt.

I'd very much like for it to just use the `children` it was handed, e.g.:

```jsx
<dl>
  <dli>
    <dt />
    <dd />
  </dli>
</dl>
```

Do you know if React will let me dictate which elements are allowed as `children`?

Reply:
Implemented. This was a great suggestion, thanks @slifty!

> Do you know if React will let me dictate which elements are allowed as children? 

Looks like no, so this just accepts any children. Ah well, the some holds true for any HTML element: if you pass it something invalid, the browser will scream at you.